src/plugins/navigation/review/navigatorTree.tpl

```diff
@@ -20,17 +20,20 @@
             </p>
         </div>
         {{else}}
-        <ul class="qti-navigator-sections collapsible-panel plain">
+        <ul aria-label="{{label}}" class="qti-navigator-sections collapsible-panel plain">
             {{#each sections}}
             <li class="qti-navigator-section collapsible {{#if active}}active{{else}}collapsed{{/if}}" data-id="{{id}}">
                 <span class="qti-navigator-label" title="{{label}}">
                     <span class="qti-navigator-text">{{label}}</span>
                     <span class="qti-navigator-counter">{{stats.answered}}/{{stats.total}}</span>
                 </span>
-                <ul class="qti-navigator-items collapsible-panel plain">
+                <ul aria-label="{{label}}" class="qti-navigator-items collapsible-panel plain">
                     {{#each items}}
                     <li class="qti-navigator-item {{cls}}" data-id="{{id}}" data-position="{{position}}">
-                        <span class="qti-navigator-label truncate" title="{{label}}">
+                        <span class="qti-navigator-label truncate" title="{{label}}"
+                              role="link" aria-disabled="{{#if viewed}}false{{else}}true{{/if}}"
+                              {{#if active}}aria-current="page"{{/if}}
+                              aria-label="{{index}} of {{../stats.questions}} {{icon}}">
```

Comment:
This will only work for actual questions. Informational items won't be taken into account.
You should rather use `stats.total` instead of `stats.questions`. Otherwise, the number of presented items won't be reflected by the screen reader announcement.